ariadne/transformations.py

```diff
@@ -457,22 +461,21 @@ class DropShort(BaseFilter):
         event_column (str, 'event' by default): Station column in data
     """
 
-    def __init__(self, num_stations=0, keep_fakes=True, station_col='station', track_col='track', event_col='event'):
+    def __init__(self, num_stations=None, keep_filtered=True, station_col='station', track_col='track', event_col='event'):
         self.num_stations = num_stations
         self.broken_tracks_ = None
         self.num_broken_tracks_ = None
-        self.filter = lambda x: x.shape[0] >= self.num_stations
-        super().__init__(self.filter, station_col=station_col, track_col=track_col,
-                         event_col=event_col, keep_fakes=keep_fakes)
+        self.filter = lambda x: len(x) >= self.num_stations
+        super().__init__(self.filter, num_stations=num_stations, station_col=station_col, track_col=track_col,
+                         event_col=event_col, keep_filtered=keep_filtered)
 
     def __repr__(self):
```

Comment:
where is the call method, where is scaler? I don't understand what is happened here

Code in master:
```python
class BaseScaler(BaseTransformer):
    """Base class for scalers.
     # Args:
         scaler (function or method pd.DataFrame -> iterable of pd.Series): scaler with fit_predict method
         columns (list or tuple, ['x', 'y', 'z'] by default): Columns to scale
         drop_old (boolean, True by default): If True, unscaled data is discarded,
                                            else preserved in columns with suffix '_old'
    """

    def __init__(self, scaler, drop_old=True, columns=('x', 'y', 'z')):
        self.columns = columns
        self.scaler = scaler
        super().__init__(drop_old=drop_old, columns=columns)

    def __call__(self, data):
        """
        # Args:
            data (pd.DataFrame):  to clean up.
        # Returns:
            data (pd.DataFrame): transformed dataframe
        """
        norms = pd.DataFrame(self.scaler.fit_transform(data[[self.columns[0], self.columns[1], self.columns[2]]]))
        data = self.transform_data(data=data, normed=norms)
        return data

    def __repr__(self):
        return '-' * 30 + '\n' + f'{self.__class__.__name__} with scaler: {self.scaler} \n' + '-' * 30 + '\n'
```